charts/csm-drivers-modules/values.yaml

```diff
@@ -0,0 +1,124 @@
+#
+#
+# Copyright © 2020-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+
+## K8S/DRIVER ATTRIBUTES
+########################
+csi-powerstore:
+  enabled: false
+  #namespace: powerstore-ns
+  version: "v2.5.0"
+  images:
+    driverRepository: dellemc
+  ## Controller ATTRIBUTES
+  controller:
+    controllerCount: 1
+    healthMonitor:
+      enabled: false
+    nodeSelector:
+    replication:
+      enabled: false
+      image: dellemc/dell-csi-replicator:v1.3.0
+    vgsnapshot:
+      enabled: false
+      image: dellemc/csi-volumegroup-snapshotter:v1.1.0
+    snapshot:
+      enabled: false
+    resizer:
+      enabled: false
+  ## Node ATTRIBUTES
+  node:
+    healthMonitor:
+      enabled: false
+    nodeSelector:
+    
+## K8S/Replication Module ATTRIBUTES
+##########################################
+csm-replication:
+  enabled: false
+  #namespace: replication-ns
+
+## K8S/Observability Module ATTRIBUTES
+##########################################
+karavi-observability:
+  enabled: false
+  #namespace: observability-ns
+  karaviMetricsPowerstore:
+    enabled: false
+  karaviMetricsPowerflex:
+    enabled: false
+  karaviMetricsPowerscale:
+    enabled: false
+
+## K8S/Application-mobility Module ATTRIBUTES
+########################################## 
+csm-application-mobility:
+  enabled: false
+  #namespace: app-ns
+  
+  # csm-application-mobility requires velero. If velero is already installed on the cluster, specify the namespace in which velero is deployed. Default value is "velero"
+  veleroNamespace: velero
+  # csm-application-mobility requires velero. If velero is not already present in cluster, set enabled to true to install it too.
+  velero:
+    enabled: false
+    credentials:
+      secretContents:
+        cloud: |
+          [default]
+          #aws_access_key_id=access_key
+          #aws_secret_access_key=secret_access_key
+    configuration:
+      provider: aws
+      backupStorageLocation:
+        name: default
+        #bucket: velero-bucket
+        config: {}
+          #region: minio
```

Comment:
Please remove values for region, s3ForcePathStyle, s3Url, publicUrl.

Reply:
Removed the values